pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -402,11 +408,12 @@ an @tech{object}, so evaluating @racket[(f 7)] starts with a
 ]
 
 Unlike in algebra, however, the @tech{value} associated with an
-argument can be changed in the body of a procedure by using
+argument parameter variable (not to be confused with parameters;
+see @secref["parameter-model"]) can be changed in the body of a procedure by using
 @racket[set!], as in the example @racket[(lambda (x) (begin (set! x 3)
-x))]. Since the @tech{value} associated with @racket[x] can be
-changed, an actual value cannot be substituted for @racket[x] when
-the procedure is applied.
+x))]. Since the @tech{value} associated with parameter variable @racket[x] can be
+changed, an argument value cannot be substituted for @racket[x] when
+the procedure is first applied.
```

Comment:
The original paragraph is clearer to me.

At least, please don't introduce "parameter variable" here ... use "argument" or "argument variable" or "formal parameter" or an existing `@tech` (if there is one).

Reply:
I discussed this with Matthew. Further up in the paragraph I introduce the term "argument parameter value" and also say not to confuse this with the (unfortunate) term "parameter." I will make sure that every instance of the term uses all three words.

Reply:
I agree that this nomenclature is confusing to me, and the old nomenclature seemed fine. Consider this a vote against this change.

Reply:
I'm not sure why people would be confused by using the term "parameter" in its absolutely standard way. I understand it's overloading the Racket term "parameter." I will switch back to "argument identifier." Would a parenthetical comment explaining why we don't use the word "parameter" be okay?

Reply:
It’s not to say that people won’t be familiar with the term “parameter” used in this sense, but rather that the overloading you mention makes it confusing which version of “parameter” is being used here, especially since the existing documentation takes great care to *only* use it to mean the dynamic construct rather than function arguments. I do think a margin note (using the `@margin-note` procedure) would be great if you want to write a sentence or two explaining the nomenclature.

Reply:
Sounds good.